index.html

```diff
@@ -275,21 +279,21 @@ <h2>Glossary</h2>
 
 <p><dfn class="lint-ignore export">Grapheme</dfn>. A character or a sequence of characters in a visual representation of some text that a typical user would perceive as being a single unit (<q>character</q>). Graphemes are important for a number of text operations such as sorting or text selection, so it is necessary to be able to compute the boundaries between each user-perceived character. For more information about graphemes and grapheme clusters, with examples, see <a href="https://www.w3.org/International/articles/definitions-characters/index#characters">Character encodings: Essential concepts</a>.</p>
           
-<p><dfn class="lint-ignore export">Grapheme cluster</dfn>. A grapheme cluster is defined by the Unicode Standard as the default mechanism for computing an approximation to [=graphemes=] (see <cite>Unicode Standard Annex #29: Text Segmentation</cite> [[UAX29]]). Two types of default grapheme cluster are defined. Unless otherwise noted, <q>grapheme cluster</q> in this document refers to an <q>extended default grapheme cluster</q>. (A discussion of grapheme clusters is also given in Section 2 of the <cite>Unicode Standard</cite>, [[Unicode]]. Cf. near the end of <a href="http://www.unicode.org/versions/latest/ch02.pdf">Section 2.11</a> in version 14.0 of The Unicode Standard.) Because different natural languages have different needs, grapheme clusters can also sometimes require tailoring. For example, a Slovak user might wish to treat the default pair of grapheme clusters "ch" as a single grapheme cluster. Note that the interaction between the language of string content and the end-user's preferences might be complex.</p>
+<p><dfn class="lint-ignore export">Grapheme cluster</dfn>. A grapheme cluster is defined by the Unicode Standard as the default mechanism for computing an approximation to [=graphemes=] (see <cite>Unicode Standard Annex #29: Text Segmentation</cite> [[UAX29]]). Two types of default grapheme cluster are defined. Unless otherwise noted, <q>grapheme cluster</q> in this document refers to an <q>extended default grapheme cluster</q>. (A discussion of grapheme clusters is also given in Section 2 of the <cite>Unicode Standard</cite>, [[Unicode]]. Cf. near the end of <a href="https://www.unicode.org/versions/latest/ch02.pdf">Section 2.11</a> in version 14.0 of The Unicode Standard.) Because different natural languages have different needs, grapheme clusters can also sometimes require tailoring. For example, a Slovak user might wish to treat the default pair of grapheme clusters "ch" as a single grapheme cluster. Note that the interaction between the language of string content and the end-user's preferences might be complex.</p>
```

Comment:
Has this changed?

Reply:
Yes, it changed the Unicode website URL from http to https.